schema.sql

```diff
@@ -1,5 +1,16 @@
 /* Database schema to keep the structure of entire database. */
 
+-- CREATE DATABASE
+-- CREATE DATABASE vet_clinic;
+-- USE vet_clinic;
+
+DROP TABLE IF EXISTS animals;
 CREATE TABLE animals (
-    name varchar(100)
+id INT GENERATED ALWAYS AS IDENTITY,
+name text NOT NULL,
```

Comment:
- [ ] (Optional) It would be better to use make the name column the type `varchar` instead of `text.` Texts are mostly used for long stings, for example, comments or descriptions while `varchar` is more appropriate for names, city locations, e.t.c.

Reply:
Thanks, I considered it but there was no size/length specification for the column.

Reply:
You can just add any number you like. 👍